docs/cloud/dev/spryker-cloud-commerce-os/troubleshooting/troubleshooting-tutorials/tutorial-troubleshooting-an-unavailable-zed.md

```diff
@@ -52,34 +52,46 @@ To check the services and tasks, do the following:
 
 {% include checking-the-status-of-ecs-services-and-tasks.md %} <!-- To edit, see /_includes/checking-the-status-of-ecs-services-and-tasks.md -->
 
+## 4. In case you have got 504 responses from the server 
```

Comment:
How about we append this section to the `## 2. Check multiple log groups at once` section? 

The information in this section refers to logs. It's quite strange that, after checking ECS services, we refer the readers back to logs.

Reply:
It is after checking ECS services because it is a specific case for a particular issue, and it should be like a stand-alone check. And you need to check warnings only if you can't find errors.